motor_msg[5] motor